src/pages/Login/ImageComponent/ImageComponent.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const ImageComponent = ({ src, alt , styleImg}) => {
```

Comment:
"styleImg" is not really for the image, but for the wrapper of the image.
So why not call it "imageWrapperStyle"?